fideslib/core/config.py

```diff
@@ -95,6 +95,9 @@ class SecuritySettings(FidesSettings):
     aes_gcm_nonce_length: int = 12
     app_encryption_key: str
     drp_jwt_secret: str
+    root_username: Optional[str] = None
```

Comment:
I would like to add a validator to make sure the username used isn't already taken but I can't find a way to do it. The database information needs to come from the installing package. For the routes we are doing this by overriding the FastAPI dependency injections, but Pydantic doesn't have the concept of dependency overrides.

Reply:
That would be a good addition. Maybe it can be addressed in a follow up instead of trying to fit it into this PR.